test/HardwareObjectsMockup.xml/mxcube-web/ui.yaml

```diff
@@ -1,146 +1,142 @@
 sample_view:
   id: sample_view
   components:
-    -
-      label: Omega
+    - label: Omega
       attribute: diffractometer.phi
       role: omega
       step: 90
       precision: 2
       suffix: °
-    -
-      label: Kappa
+    - label: Kappa
       attribute: diffractometer.kappa
       role: kappa
       step: 0.1
       precision: 1
       suffix: "°"
-    -
-      label: Kappa Phi
+    - label: Kappa Phi
       attribute: diffractometer.kappa_phi
       role: kappa_phi
       step: 0.1
       precision: 1
       suffix: °
-    -
-      label: X
+    - label: X
       attribute: diffractometer.phix
       role: phix
       step: 0.1
       precision: 3
       suffix: mm
-    -
-      label: Y
+    - label: Y
       attribute: diffractometer.phiy
       role: phiy
       step: 0.1
       precision: 3
       suffix: mm
-    -
-      label: Z
+    - label: Z
       attribute: diffractometer.phiz
       role: phiz
       step: 0.1
       precision: 3
       suffix: mm
-    -
-      label: Samp-X
+    - label: Samp-X
       attribute: diffractometer.sampx
       role: sampx
       step: 0.1
       precision: 3
       suffix: mm
-    -
-      label: Samp-Y
+    - label: Samp-Y
       attribute: diffractometer.sampy
       role: sampy
       step: 0.1
       precision: 3
       suffix: mm
-    -
-      label: Sample Horizontal
+    - label: Sample Horizontal
       attribute: diffractometer.phiz
       role: sample_horizontal
       step: 0.1
       precision: 3
       suffix: mm
-    -
-      label: Sample Vertical
+    - label: Sample Vertical
       attribute: diffractometer.phiy
       role: sample_vertical
       step: 0.1
       precision: 3
       suffix: mm
-    -
-      label: Zoom
+    - label: Zoom
       attribute: diffractometer.zoom
       role: zoom
       step: 1
       precision: 1
       suffix: mm
-    -
-      label: Focus
+    - label: Focus
       attribute: diffractometer.sampx
       role: focus
       step: 0.1
       precision: 3
       suffix: mm
 
+sample_view_video_controls:
```

Comment:
The option to display the focus control on the video display was moved to `ui.yml` and the same option was added to all controls (in case it would be needed)

Reply:
are you thinking on potentially hide the 3 click, zoom. etc?

Reply:
Exactly, I thought it was better to simply generalize the visibility of these controls, as there already was one that could be hidden/shown. Its less likely for some controls but it is useful for a few especially for SSX